app/components/inventory/rank-select.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+import SelectValues from 'hospitalrun/utils/select-values';
+
+const c = Ember.computed;
+
+export default Ember.Component.extend({
+  label: 'Rank',
+  rankOptions: [],
```

Comment:
Imho setting this in initializer is little bit over-kill and made code confusing. Setting this to undefined/null by default do not specify type (maybe it will be better to remove this and keep only setter in init). The only reason I kept this is to let everyone know that this property will be and it should be `Array`. Computed property seems to be best solution but I'm not sure if there is no any reasonable overhead (since this type implements little bit more than we actually need). ES7 solution seems to be best for me, but I'm not sure, if it's compatible.

I can remove this completely (but for me it makes sense to have defined all props that instance has on one place) and keep the setter in `init()` or make CP (if it is not too much expensive)

I do not know as much about `ember-metal` as you so let me know your suggestions on this.